ServerCore/Areas/Identity/Pages/Account/Edit.cshtml

```diff
@@ -0,0 +1,62 @@
+﻿@page
+@model EditModel
+@{
+    ViewData["Title"] = "Edit User Properties";
+}
+
+<h2>@ViewData["Title"]</h2>
+<h4>Edit Properties for @Model.PuzzleUser.Name</h4>
+<hr />
+
+<p class="text-info">
+    This profile is shared between all events.
+</p>
+
+<div class="row">
+    <div class="col-md-4">
+        <form method="post" asp-route-returnUrl="@Model.ReturnUrl">
+            <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+            <input type="hidden" asp-for="PuzzleUser.ID" />
+            <input type="hidden" asp-for="PuzzleUser.IdentityUserId" />
```

Comment:
Please validate that this hasn't been tampered with

Reply:
I don't understand what this comment means.

Reply:
Name-dropping @morganbr  so he sees this.

Reply:
Sitting next to Morgan - comment withdrawn since this is an admin only page.